src/utils/utils.ts

```diff
@@ -1,7 +1,11 @@
+import { CartItem } from "../types";
+
 export const delay = (time: number): Promise<true> => {
   return new Promise((resolve) => {
     setTimeout(() => {
       resolve(true);
     }, time)
   })
-}
\ No newline at end of file
+}
+
+export const totalCartPrice = (cart: CartItem[]) => cart.reduce((total, item) => (total += item.bookPrice), 0);
```

Comment:
This fn should be inside the domain/cart directory

Reply:
Adding these changes for this